tests/correctness/systolic/static/array-8.expect

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
Just wondering if we should have the same `.expect` file for the dynamic and static array to ensure they are producing the same value? The tricky thing is the `cycles` field but we can remove it using `jq`.

Reply:
I think it would be nice to do this: one runt question I'm having trouble with: is there a way for two different input files to have the same ".expect" and ".data" file?

Reply:
Yeah, the trick is using the `expect_dir` location for the test suite so that the `.expect` files are generated in the same place. For example, here is one for a Filament test suite: https://github.com/cucapra/filament/blob/main/runt.toml#L82

Reply:
Thanks